configure.py

```diff
@@ -444,8 +444,6 @@ def process_command_line(args): # pylint: disable=too-many-locals
 
     install_group.add_option('--prefix', metavar='DIR',
                              help='set the install prefix')
-    install_group.add_option('--destdir', metavar='DIR',
```

Comment:
Could you throw a UserError that helps users what to do instead of using `--destdir`? See for example "--via-amalgamation" and "--gen-amalgamation" in the configure script